library/src/G4CMPCrystalGroup.cc

```diff
@@ -10,7 +10,8 @@
 //
 // 20170728  Change function args "alpha, beta, gamma" to "al, bt, gm" (-Wshadow)
 // 20240426  S. Zatschler -- Add explicit fallthrough statements to switch cases
-// 20240924  I. Hernandez -- Add an if statement to include a rhombohedral Crystal group and filling more explicitly the function G4CMPCrystalGroup::FillOrthorhombic()
+// 20241014  I. Hernandez -- Add an if statement to include a rhombohedral Crystal group.
+//           Filling more explicitly the function G4CMPCrystalGroup::FillOrthorhombic()
```

Comment:
This line is still longer than 80 characters.  Does your editor support specifying a column after which to automatically insert a line break?

Reply:
I added the following line. I decided to break it into two comments because this is how I solved this bug. 
First,  I added the if statement, and later I fixed the function  FillOrthorhombic()
// 20241024  I. Hernandez -- Add an if statement to include a rhombohedral Crystal group
// 20241024  I. Hernandez -- Fixing the function FillOrthorhombic()